frontend/src/metabase/visualizations/components/ScalarValue/ScalarValue.jsx

```diff
@@ -23,54 +22,25 @@ import {
   ScalarTitleContent,
   ScalarValueWrapper,
 } from "./ScalarValue.styled";
-import { findSize, getMaxFontSize } from "./utils";
 
 export const ScalarWrapper = ({ children }) => (
-  <ScalarRoot>{children}</ScalarRoot>
+  <ScalarRoot className={DashboardS.ScalarValueContainer}>
+    {children}
+  </ScalarRoot>
 );
 
-const ScalarValue = ({
-  value,
-  height,
-  width,
-  gridSize,
-  totalNumGridCols,
-  fontFamily,
-}) => {
+const ScalarValue = ({ value }) => {
   const {
     other: { number: numberTheme },
   } = useMantineTheme();
 
-  const fontSize = useMemo(() => {
-    if (numberTheme?.value?.fontSize) {
-      return numberTheme.value?.fontSize;
-    }
```

Comment:
This might still be needed for the theme overrides, FYI.

Reply:
ah nice call